sdks/go/container/boot.go

```diff
@@ -123,7 +123,7 @@ func main() {
 		"--options=" + options,
 	}
 	if info.GetStatusEndpoint() != nil {
-		args = append(args, "--status_endpoint=" + info.GetStatusEndpoint().GetUrl())
```

Comment:
This seemse like an unrelated change. Please remove if so.

Reply:
This was applied by running the linter with too wide a scope. resolved.